packages/@functionless/ast/src/ensure.ts

```diff
@@ -69,7 +69,7 @@ export function ensure<Assert extends Assertion>(
         .map((kind) =>
           typeof kind === "number" ? getNodeKindName(kind) : kind
         )
-        .join(", ")}`
+        .join(", ")}, found: ${item}`
```

Comment:
Will this print an object usefully?

Reply:
Not always, but it did when I was testing. Will stringify.